db/db_impl.cc

```diff
@@ -1843,6 +1846,23 @@ void DBImpl::MultiGetImpl(
   PERF_TIMER_STOP(get_post_process_time);
 }
 
+void DBImpl::SampleMemTableReadAndMaybeFlush(ColumnFamilyData* cfd, MemTable* mem) {
```

Comment:
I'm not against having a functionality to trigger flush, but I hope we are careful about it. We don't want to trigger flush for the DB that still has reasonable write rate into it.